docs/Userguide_sharing_data.rst

```diff
@@ -0,0 +1,101 @@
+Sharing Data with ACLs
+======================
+
+As an illustrative example, to allow ``$USER`` to share with ``$USER2`` in
+``rwx`` a hierarchy ``/network/scratch/$USER/X/Y/Z/...`` with ACLs:
+
+----
+
+| Grant **oneself** permissions to access any **future** files/folders created
+  by the other *(or oneself)*
+| (``-d`` renders this permission a "default" / inheritable one)
+
+.. code-block:: bash
+
+   setfacl -Rdm user:$USER:rwx  /network/scratch/$USER/X/Y/Z/
```

Comment:
I'm guessing `-R` means recursive? `-d` is explained, what about `-m`?

Reply:
Yes, `-R` is recursive. `-m` means "modify" (add that ACL rule to the set), as opposed to "set" (replace all ACL rules with the ones provided).

More on that in `man setfacl`. I think people should develop at least some reflex to consult the man-pages, it's almost always `man commandname`.

Reply:
People need to use the man-pages to get further explanation about commands and parameters. The documentation emphasizes the important parameters in the context, but we should not have to duplicate the man pages by explaining each parameter.